discovery-provider/src/schema_track_simple_oneof.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
I'm a bit unclear on the relationship between this one and the other one? How do these get used together? Maybe add a comment in the validator file?

Reply:
`schema_track_simple.json` validates against a track object and its entirety, while `schema_track_simple_oneof.json` validates against one field (at the moment, either `download` or `title`). for our purposes in using the sqlalchemy `@validates`, we would use the latter schema. 

we _could_ use the first schema to validate an object in the indexing flow, and the second schema to validate a field in models.py. however, i figure using the latter is enough. ill go ahead and remove the first schema in my next commit

Reply:
Ideally we could have one schema where we would define the entire json object and choose to validate a single key or everything at once. I don't think that's possible w/ json schema w/o doing additional work. So if it we're sticking w/ this `oneof` file, we would have to map over each key and validate it right?

Reply:
if we wanted to validate against an entire object, then yes, we would need to map over each key and validate each field with the `oneOf` schema.

and yes, you're correct. we would need to figure out a way to have one schema do an entire json object validation or each field validation. what we could do now for easy fix is keep both schemas to validate against each field and against the entire object itself. this would be much quicker since both schemas currently work